algosdk/transaction.py

```diff
@@ -10,19 +10,47 @@
 from nacl.exceptions import BadSignatureError
 
 
+class SuggestedParams:
```

Comment:
Could this instead be `Params` or `TransactionParams`? I feel like they're only `suggested` when they come from algod, which isn't always the case. Or phrased another way, I wonder if it will be weird to users to construct a `SuggestedParams` themselves, especially when function signatures refer to them as only coming from algod, or if they'll wonder if their user-set params are only a suggestion.

Reply:
NVM I see this was discussed already :+1: :+1: